components/greaselion/browser/BUILD.gn

```diff
@@ -16,9 +14,11 @@ source_set("browser") {
   deps = [
     "//base",
     "//brave/components/brave_component_updater/browser",
+    "//chrome/browser/extensions:extensions",
+    "//chrome/common",
```

Comment:
I think we can also delete `//chrome/common`.

Reply:
Looks like greaselion_service_impl.cc has just been including chrome/common/chrome_paths.h without uses. Removed.